karmagrambot/__init__.py

```diff
@@ -8,6 +8,12 @@
 
 logging.basicConfig()
 
+def check_message(message):
```

Comment:
1: Both `if` and `elif` check the same condition, therefore `elif` never runs.
2: What happens if the message is not composed of +s or -s? (tip: this returns `None` which will evaluate to `False` and count as a downvote).